src/content/peerconnection/create-offer/js/main.js

```diff
@@ -49,7 +48,8 @@ function createOffer() {
     }]
   };
   // These constraints confuse Firefox, even if declared as optional.
-  if (webrtcDetectedBrowser !== 'Firefox') {
+  // See https://bugzilla.mozilla.org/show_bug.cgi?id=1184712
```

Comment:
On second thought, this is createOffer, so there shouldn't be a problem. Are we sure this is needed?